### DISCUSSION TITLE
Bulk download imagery from WMS (Where permitted)?

### DISCUSSION THREAD

**Discussion by nebsar** [pelicanmapping/osgearth#1946] [Support Q&A] [answered]
Is it possible to bulk downloading and indexing the map tiles, where bulk download is permitted?

Thanks

**Comment by gwaldron** [comment, marked as answer]
Bulk downloading from a WMS server is rarely permitted. If you want local data you should contact the service provider and ask them about it.

**Reply by nebsar** [reply]
Thank you Glenn.
